packages/components/transfer/src/index.vue

```diff
@@ -183,14 +185,14 @@ export default defineComponent({
       emit
     )
 
-    const leftPanel = ref(null)
-    const rightPanel = ref(null)
+    const leftPanel = ref<TransferType | null>(null)
+    const rightPanel = ref<TransferType | null>(null)
```

Comment:
may be `const left Panel = ref<TransferType>()` will be better

Reply:
Updated